deeprankcore/Trainer.py

```diff
@@ -183,10 +210,10 @@ def load_model(self, dataset_train, dataset_val, dataset_test, Net):
         if self.cluster_nodes is not None:
             if self.cluster_nodes in ('mcl', 'louvain'):
```

Comment:
Wouldn't it be better to replace such input parameters by enumeration objects?

Reply:
I know enumeration in Python but I'm not getting what you mean. An example for the above code?